homeassistant/components/litterrobot/quality_scale.yaml

```diff
@@ -0,0 +1,86 @@
+rules:
```

Comment:
`binary_sensor.py` -> Personal preference since it increases consistency

the order of the file is usually

[entity description classes]
[entity descriptions]
[async_setup_entry]
[entity classes]